plugins/poll/plugin.rb

```diff
@@ -268,6 +268,29 @@ def voters
     true
   end
 
+  NewPostManager.add_handler(1) do |manager|
```

Comment:
We have to validate before the `QueuedPost` record is created otherwise we run the risk of having a post that fails validation being stucked in the queue.